### ISSUE TITLE
VCH doesnt bind to container IO stream

### ISSUE THREAD

**Issue by smaithani** [vmware/vic#2839]
Running the following command just exits and drops me to my mac shell instead of connecting to the container IO stream
[container-logs.zip](https://github.com/vmware/vic/files/543182/container-logs.zip)
:

> > docker -H 10.161.19.210  run -it --name mybusybox busybox

**Comment by mdubya66** [comment]
@smaithani are you sure it wasn't the busybox shell? can you paste the shell?

```(mwilliamson) dubs:~ $ docker -H 172.16.127.193:2376 --tls run -it --name mybusybox busybox
/ #```

**Comment by smaithani** [comment]
100% sure it was not busybox shell

**Comment by smaithani** [comment]
Happened once again today, this issue is pretty frequent

**Comment by mdubya66** [comment]
@hickeng please take a look at the logs. I didn't see anything that stood out.

**Comment by smaithani** [comment]
Just an update, I keep running into this on a daily basis.

**Comment by hickeng** [comment]
@smaithani Please could you try with 0.7.0 - there's been significant work done on the attach paths which will likely have addressed this problem.

My speculation as to the issue is that this environment is running extremely slowly and the timeout we put in for attach to protect people from hung attach sessions when trying to connect to a container that's transitioning to off is kicking in:

initial port layer call for the attach:
```
time="2016-10-20T23:25:39Z" level=debug msg="Attempting to get ssh session for container 25f10cbee52f2e4a7df3791ca71d420743ced5214e240a226135743a6c713079 stdout with deadline Thu Oct 20 23:26:09 UTC 2016"
```
```
time="2016-10-20T23:25:45Z" level=debug msg="Found 1 subscribers to vsphere.VMEvent: mybusybox-25f10cbee52f2e4a7df3791ca71d420743ced5214e240a226135743a6c713079 on  sc-rdops-vm09-dhcp-3-202.eng.vmware.com in vcqaDC is powered on"
```
```
time="2016-10-20T23:26:19Z" level=debug msg="Found 1 subscribers to events.ContainerEvent: Container 25f10cbee52f2e4a7df3791ca71d420743ced5214e240a226135743a6c713079 Started"
```
and the port layer call that would have completed the attach, 38s later.
```
time="2016-10-20T23:26:19Z" level=debug msg="[ END ] [github.com/vmware/vic/lib/portlayer/attach.(*Server).Get:91] [77.126µs] 25f10cbee52f2e4a7df3791ca71d420743ced5214e240a226135743a6c713079" 
```

**Comment by mlh78750** [comment]
@smaithani : This should be resolved in 0.7. Please let us know if you are still seeing this issue.

**Comment by mlh78750** [comment]
@smaithani I am going to close this as no-repo since we believe this is resolved in the 0.7 or newer build. If you are still running into this issue please let us know and we will reopen the issue and sort it out.
